src/map/script.cpp

```diff
@@ -5087,6 +5087,21 @@ BUILDIN_FUNC(next)
 	return SCRIPT_CMD_SUCCESS;
 }
 
+/// Clear The dialog text and continues the script without Next button.
+///
+/// clear;
+BUILDIN_FUNC(clear)
+{
+	TBL_PC* sd;
+
+	if (!script_rid2sd(sd))
+		return SCRIPT_CMD_SUCCESS;
+
+	st->state = RUN;
```

Comment:
Is changing the state really necessary?
It should be running anyway. Otherwise it would not reach the statement.